### ISSUE TITLE
jitpack problem

### ISSUE THREAD

**Issue by mgeier63** [rehacktive/waspdb#5]
>  compile 'com.github.rehacktive:waspdb:1.0'

doesn't work for me. This retrieves the example app but I do not want  to depend on the example app.

As per jitpack docs, the following should work to retrieve the "waspdb" library submodule

>  compile 'com.github.rehacktive.waspdb:waspdb:1.0'

But unfortunately then gradle can't find it. 
Maybe you need to tweak you jitpack config a bit to allow depending on the library only?

**Comment by rehacktive** [comment]
Did you also add this to the main build gradle file?

```
allprojects {
    repositories {
        ...
        maven { url "https://jitpack.io" }
    }
}

```

**Comment by mgeier63** [comment]
sure i did, and

> compile 'com.github.rehacktive:waspdb:1.0'

resolves fine, however it's the wrong thing. I'd like to reference the library project only!

See here for an example of how it is supposed to work with submodules

> https://github.com/jitpack/gradle-modular

But for some reason, when I try to apply this logic to your project, i.e.

> compile 'com.github.rehacktive.waspdb:waspdb:1.0'

it doesn't resolve.

**Comment by rehacktive** [comment]
Investigating.

**Comment by mgeier63** [comment]
As a workaround you could consider splitting this into two repositories, one for the core library and one for the example app

**Comment by rehacktive** [comment]
So I've read around. Doing the submodule is not suggested.
Btw, I've tried to import simply as suggested (compile 'com.github.rehacktive:waspdb:1.0') and I don't see any class imported from the sample app, only classes from waspdb library.
So where's the problem? 

I'll be happy to fix, but please help me to understand what's the issue.

**Comment by mgeier63** [comment]
Hmm,  it seems it's working fine now;-) 

To clarfiy,
yesterday I started exactly like suggested and It didn't work. All I got was the (generated) "R" class of your package, no other classes, neither from the library not form the example app.

Only then did I start to toy around with the submodule stuff, 'cause I thought that might be the reason and/or it might fix it.

But since it's working today, most likely this was (again) a temporary glitch at the jitpack side.[Or the stupid android studio gradle integration not properly cleaning up stuff]

Thank you for your time and sorry for the confusion!
